app/task/project/loadotio/loadotio.cpp

```diff
@@ -88,11 +89,51 @@ bool LoadOTIOTask::Run()
 
   // Keep track of imported footage
   QMap<QString, Footage*> imported_footage;
+  QMap<OTIO::Timeline*, Sequence*> timeline_sequnce_map;
 
+  // Variables used for loading bar
+  float number_of_clips = 0;
+  float clips_done = 0;
+
+  // Generate a list of sequences with the same names as the timelines.
+  // Assumes each timeline has a unique name.
+  int unnamedSequenceCount = 1;
   foreach (auto timeline, timelines) {
-    // Create sequence
     Sequence* sequence = new Sequence();
-    sequence->SetLabel(QString::fromStdString(timeline->name()));
+    if (!timeline->name().empty()) {
+      sequence->SetLabel(QString::fromStdString(timeline->name()));
+    } else {
+      // If the otio timeline does not provide a name, create a default one here
+      QString label = "Sequence " + QString::number(unnamedSequenceCount);
```

Comment:
Should use `tr("Sequence %1").arg(unnamedSequenceCount)` so the string can be translated correctly. Also, code style requires underscored variables rather than camel-case (i.e. `unnamed_sequence_count`)

Reply:
Sorry, can't believe I just did that (missed the `tr()` that is)